examples/sbol2/CreatingSBOL2Objects/Cut.ipynb

```diff
@@ -0,0 +1,93 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sbol2\n",
+    "\n",
+    "# Create an SBOL document\n",
+    "doc = sbol2.Document()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set a namespace for the document\n",
+    "sbol2.setHomespace('https://github.com/SynBioDex/SBOL-Notebooks')\n",
+    "\n",
+    "# Create a Sequence object\n",
+    "seq = sbol2.Sequence('example_sequence')\n",
+    "seq.elements = 'ATGCGTACGTAGCTAGTCTGATCGTAGCTAGT'\n",
+    "seq.encoding = sbol2.SBOL_ENCODING_IUPAC\n",
+    "\n",
+    "# Add the sequence to the document\n",
+    "doc.addSequence(seq)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create a ComponentDefinition object for a DNA component\n",
+    "comp_def = sbol2.ComponentDefinition('example_component', sbol2.BIOPAX_DNA)\n",
+    "comp_def.sequences = [seq.persistentIdentity]\n",
+    "\n",
+    "# Add the ComponentDefinition to the document\n",
+    "doc.addComponentDefinition(comp_def)\n",
+    "\n",
+    "# Create a SequenceAnnotation object to describe a notable feature\n",
+    "seq_annotation = sbol2.SequenceAnnotation('cut_feature')\n",
+    "\n",
+    "# Create a Cut Location\n",
+    "cut = sbol2.Cut('cut_location')\n",
+    "cut.at = 10  # Define where the cut is located (position 10)\n",
+    "cut.orientation = sbol2.SBOL_ORIENTATION_INLINE\n",
+    "\n",
+    "# Add the Cut location to the SequenceAnnotation\n",
+    "seq_annotation.locations.add(cut)\n",
+    "\n",
+    "# Add the SequenceAnnotation to the ComponentDefinition\n",
+    "comp_def.sequenceAnnotations.add(seq_annotation)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Save the document to an SBOL file\n",
```

Comment:
Document should be validated before it's written, to confirm its contents are correct

Reply:
Thanks - I've added a line for this.